app.js

```diff
@@ -12,6 +13,10 @@ const app = express();
 /**
  * Middleware
  */
+app.use(cors({
+  origin: ['http://localhost:3000'],
+  credentials: true,
+}));
```

Comment:
Should this be retrieved from config instead for the day in the future when we want this on an actual remote server?

Reply:
Yes. I'll abstract it.